Source/items.cpp

```diff
@@ -3005,7 +3005,7 @@ void CalcPlrInv(int playerId, bool loadgfx)
 		CalcPlrBookVals(player);
 		player.CalcScrolls();
 		CalcPlrStaff(player);
-		if (playerId == MyPlayerId && currlevel == 0)
+		if (currlevel == 0)

```

Comment:
This seams a bit unrelated to manashield?

Reply:
ye but I had some code there earlier and cleaned this useless if up